packages-exp/auth-compat-exp/index.ts

```diff
@@ -140,5 +139,4 @@ function registerAuthCompat(instance: _FirebaseNamespace): void {
   instance.registerVersion('auth', version);
 }
 
-impl.registerAuth(_getClientPlatform());
```

Comment:
What happened with this, did getClientPlatform turn out to be unnecessary?

Reply:
Auth-exp will register itself, so it's not necessary to do it in auth-compat